ais/api/views.py

```diff
@@ -395,18 +477,14 @@ def search_view(query):
                 'include_units' in request.args,
                 is_range=high_num is not None,
                 is_unit=unit_type is not None) \
-                .exclude_non_opa('opa_only' in request.args)
+                .exclude_non_opa('opa_only' in request.args) \
```

Comment:
ok sorry, i didn't see the `elif`. i think all `/search` needs to be is a router to other view functions, like:

    parser_search_type_map = {
        'address': 'address',
        'intersection_addr': 'intersection',
        ...
    }
    parsed = parser.parse(input)
    search_type = parsed['type']
    # get the corresponding view function
    view = globals()[search_type]
    # call it
    view(query)

the only catch here is there may be a small amount of parsing/scrubbing repeated, in which case we could always add a kwarg to the individual view functions like `already_normalized=False` to keep that from happening.